.github/workflows/codeql.yml

```diff
@@ -1,12 +1,12 @@
-name: "CodeQL"
+name: 'CodeQL'
```

Comment:
**issue:** can we remove the changes to this file, since it's not part of the suggested changes?

Reply:
fixed